backend/private-graph/graph/schema.resolvers.go

```diff
@@ -8783,10 +8784,13 @@ And specifically, for the %s product, you can refer to the following documentati
 			request:  "logs that have 'panic' in the message",
 			response: `{"query":"message=*panic*","date_range":{"start_date":"","end_date":""}}`,
 		},
+		{
+			request:  openai_interface.IrrelevantQuery,
+			response: `{"query":"","date_range":{"start_date":"","end_date":""}}`,
+		},
 	}
 
-	resp, err := client.CreateChatCompletion(
-		context.Background(),
+	resp, err := r.OpenAiInterface.CreateChatCompletion(
```

Comment:
Seeing a `"openai client is nil"` error from hitting this resolver, not sure if it may be a local issue